### DISCUSSION TITLE
Why Prisma there's no a native softDelete?

### DISCUSSION THREAD

**Discussion by LeandroVCastro** [prisma/prisma#21704] [Q&A] [answered]
### Question

I'm used to handle Eloquent ORM (PHP/Laravel), and I'm missing softDelete resource os Prisma.

I know this is possible using a middleware.

But this is a very trivial resource.

Then, why Prisma there's no a native softDelete method?

**Comment by ludralph** [comment, marked as answer]
Hi @LeandroVCastro 👋 

Thank you for raising this question.

We do have an open [feature request](https://github.com/prisma/prisma/issues/3398) to add soft delete to the core of Prisma. Also, based on this [comment](https://github.com/prisma/prisma/issues/3398#issuecomment-1374005812) and i quote: 

> We will certainly look into offering soft delete functionality either directly in Prisma, or via a Prisma Client Extension or help someone who wants to do that. Right now this is not a focus though, so wait until we post an update here when we decide to tackle this.

We do have this in our sights but just have not prioritized it yet.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by Color-Kat** [reply]
still no?
